packages/react/src/components/Shimmer/Shimmer.types.ts

```diff
@@ -53,6 +53,11 @@ export interface IShimmerProps extends React.AllHTMLAttributes<HTMLElement>, Rea
    * Defines an object with possible colors to pass for Shimmer customization used on different backgrounds.
    */
   shimmerColors?: IShimmerColors;
+
+  /**
+   * Only use if `customElementsGroup` has a single parent `div`. If a custom element is not provided, defaults to true.
+   */
+  improveCSSPerformance?: boolean;
```

Comment:
Would be awesome if this could default to being enabled.

Reply:
Agreed but I think that would be a breaking change. In most cases this should end up being enabled. In cases where `customElementsGroup` is used we cannot be sure the root element is a `div` so we need to fall back to the previous selector in that case and users will have to opt in to this new selector.